index.bs

```diff
@@ -259,6 +262,46 @@ the user agent SHOULD run these steps:
 2. If |video| is {{pictureInPictureElement}}, run the <a>exit
     Picture-in-Picture algorithm</a>.
 
+## Auto Picture-in-Picture ## {#auto-pip}
+
+<em>The paragraph below is non-normative.</em>
+
+Some pages may want to automatically enter and leave Picture-in-Picture for
+a video element; for example, video meetings web apps would benefit from
+some automatic Picture-in-Picture behavior when user switches back and forth
+between the web app and other applications/tabs.
+To support these use cases, a new {{autoPictureInPicture}} attribute is
+added to the list of content attributes for video elements.
+
+The {{autoPictureInPicture}} IDL attribute MUST <a>reflect</a> the content
+attribute of the same name.
+
+The <dfn>autoPictureInPictureElement</dfn> is the video element, among all
+video elements with the {{autoPictureInPicture}} attribute currently set,
+whose {{autoPictureInPicture}} attribute was set most recently. Note that the
```

Comment:
I think _was set most recently_ would benefit from a clear definition. Happy to have this dealt with in a follow-up though.

Reply:
It is tracked at https://github.com/WICG/picture-in-picture/issues/115